<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteForm</class>
 <widget class="QWidget" name="DeleteForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>281</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Пациент</string>
    </attribute>
    <widget class="QPushButton" name="patientDeleteButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Удалить запись</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="patientIDedit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="patientLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ID пациента</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Приём</string>
    </attribute>
    <widget class="QPushButton" name="appointmentDeleteButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Удалить запись</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="appointmentIDedit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="appointmentLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ID приёма</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Врач</string>
    </attribute>
    <widget class="QPushButton" name="doctorDeleteButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Удалить запись</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="doctorEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="doctorLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Фамилия врача</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
